assets/src/dashboard/components/cardGallery/index.js

```diff
@@ -124,23 +110,28 @@ function CardGallery({ children }) {
             rowHeight={metrics.miniWrapperSize.height}
             gap={metrics.gap}
           >
-            {cards.map((card, index) => (
+            {pages.map((page, index) => (
               <MiniCardWrapper
                 key={index}
-                isSelected={index === activeCardIndex}
+                isSelected={index === activePageIndex}
                 {...metrics.miniWrapperSize}
                 onClick={() => handleMiniCardClick(index)}
               >
-                <MiniCard {...metrics.miniCardSize}>{card}</MiniCard>
+                <MiniCard {...metrics.miniCardSize}>
+                  <PreviewPage page={page} />
+                </MiniCard>
               </MiniCardWrapper>
             ))}
           </MiniCardsContainer>
         </UnitsProvider>
       )}
-      {metrics.activeCardSize && cards[activeCardIndex] && (
+      {metrics.activeCardSize && pages[activePageIndex] && (
         <UnitsProvider pageSize={metrics.activeCardSize}>
           <ActiveCard {...metrics.activeCardSize}>
-            {cards[activeCardIndex]}
+            <PreviewPage
+              page={pages[activePageIndex]}
+              animationState={STORY_PAGE_STATE.ANIMATE}
+            />
```

Comment:
I think this refactor is good but another way to do it could have been to create a `onCardSelected` callback that passes back the `activeIndex` of the selected card.  Then `details` or whomever can choose to do whatever they want with the active item based on index.  It would keep things more generic, but considering this component will only ever be used for stories that's probably overkill :D and this refactor keeps things much cleaner!!